.github/workflows/nightly-builds.yml

```diff
@@ -3,6 +3,7 @@ name: Nightly Builds
 on:
   schedule:
     - cron: "0 0 * * *"
+  workflow_dispatch:
```

Comment:
Do I understand correctly that this means you *can* trigger this workflow manually or from another workflow, but we're not doing that anywhere automatically yet?

Reply:
Yes, mostly to be able to trigger-test changes manually, not so much for the trigger-from-another-workflow aspect